modules/otel-troubleshooting-instrumentation-injection-into-your-workload.adoc

```diff
@@ -0,0 +1,76 @@
+// Module included in the following assemblies:
+//
+// * observability/otel/otel-troubleshooting.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="otel-troubleshooting-instrumentation-injection-into-your-workload_{context}"]
+= Troubleshooting instrumentation injection into your workload
+
+Troubleshooting instrumentation injection into your workload involves checking whether the `Instrumentation` object was created, the init-container started, the resources were deployed in the correct order, looking for errors in the operator logs, and double-checking the pod annotations.
+
+.Procedure
+
+. Run the following command to verify that the `Instrumentation` object was successfully created:
++
+[source,console]
+----
+oc get instrumentation -n <workload_project> # <1>
+----
+<1> The namespace where the instrumentation is injected for workloads.
```

Comment:
@iblancasa, looks like I didn't yet update this line in response to your suggestion below: 
```suggestion
<1> The namespace where the instrumentation was created.
```